From 59efae8920ff919beffa3816874c2e7cc85f3f08 Mon Sep 17 00:00:00 2001
From: Jasper St. Pierre <jstpierre@mecheye.net>
Date: Wed, 25 May 2011 20:47:53 -0400
Subject: [PATCH] Fix XShape

Commit c3a04bf unintentionally broke XShape handling. By studying the code
extremely carefully, I found this inconsistency with the code that was
there before.

https://bugzilla.gnome.org/show_bug.cgi?id=635268
---
 src/ui/frames.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/ui/frames.c b/src/ui/frames.c
index 9f0126c..4352e38 100644
--- a/src/ui/frames.c
+++ b/src/ui/frames.c
@@ -880,7 +880,7 @@ apply_cairo_region_to_window (Display        *display,
 
   XShapeCombineRectangles (display, xwindow,
                            ShapeBounding, 0, 0, rects, n_rects,
-                           ShapeSet, YXBanded);
+                           op, YXBanded);
 
   g_free (rects);
 }
-- 
1.7.5.1

